components/automate-ui/src/app/page-components/navbar/navbar.component.html

```diff
@@ -35,6 +35,7 @@
       </div>
     </div>
     <div class="right-nav" role="menubar">
+      <app-projects-filter></app-projects-filter>
```

Comment:
In order to hide this behind the beta 2.1 flag, you'll want to add an IAM minor version check.
https://github.com/chef/automate/blob/master/components/automate-ui/src/app/pages/project/list/project-list.component.ts#L53
https://github.com/chef/automate/blob/master/components/automate-ui/src/app/pages/project/list/project-list.component.html#L16

Reply:
👍 Pushed a commit that adds a check for the 2.1 flag:  https://github.com/chef/automate/pull/172/commits/e0dbabdeabf6a4c8cf7ba83aa3bd755a62b16a21

It seems the iam version isn't fetched until you navigate to the Settings view though. We'll need to make that happen on app load instead.

![](https://user-images.githubusercontent.com/479121/56768668-397af880-676c-11e9-9983-37f063afad66.gif)

Reply:
would that mean adding a call to ui.component.ts?